From d2f3053e156a8dfc3e2e7f8fa814f671d851b26f Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 4 Jul 2014 08:15:21 +0000
Subject: [PATCH] Include sys/select.h for FD_SETSIZE

This fixes the following compile error with musl libc:
watchgnupg.c:225:18: error: 'FD_SETSIZE' undeclared (first use in this
function)
   else if (fd >= FD_SETSIZE)
                  ^

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 tools/watchgnupg.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/watchgnupg.c b/tools/watchgnupg.c
index 4f4d54d..7aede22 100644
--- a/tools/watchgnupg.c
+++ b/tools/watchgnupg.c
@@ -28,6 +28,7 @@
 #include <stdarg.h>
 #include <assert.h>
 #include <unistd.h>
+#include <sys/select.h>
 #include <sys/socket.h>
 #include <sys/un.h>
 #include <netinet/in.h>
-- 
2.0.1

